server/package.json

```diff
@@ -9,7 +9,8 @@
     "start": "node compiled/app.js",
     "dev": "nodemon --watch 'src/**/*.ts' --exec 'ts-node' src/app.ts",
     "pretest": "eslint . --ext .ts --fix",
-    "test": "cross-env NODE_ENV=test mocha -r ts-node/register 'tests/**/*.ts' --timeout 120000 --exit",
+    "test": "cross-env NODE_ENV=test mocha -r ts-node/register 'tests/**/*.ts' --timeout 24000 --exit",
+    "coverage": "nyc npm test",
```

Comment:
This outputs a coverage report, maybe we can upload it to codecov.io in a later PR.